src/components/ui/SearchBox/SuggestionList.tsx

```diff
@@ -0,0 +1,231 @@
+			{isLoading && <li className="autosuggest-menu__item loading-label">Loading...</li>}
+			{hoverProducts && hoverProducts.length ? (
+				<ProductsComponent
+					products={hoverProducts}
+					ProductHeading={ProductHeading}
+					getItemProps={getItemProps}
+					redirectItemDetails={redirectItemDetails}
+					isAtleastOneExist={isAtleastOneExist}
+					onViewMatches={onViewMatches}
+					downshift={downshift}
+					searchedKeyword={searchedKeyword}
+				/>
+			) : (
+					<ProductsComponent
+						products={products}
+						ProductHeading={ProductHeading}
+						getItemProps={getItemProps}
+						redirectItemDetails={redirectItemDetails}
+						isAtleastOneExist={isAtleastOneExist}
+						onViewMatches={onViewMatches}
+						downshift={downshift}
+						searchedKeyword={searchedKeyword}
+					/>
+				)}
+			<div>
+				{categories && categories.length ? (
+					<>
+						<h3>{CategoryHeading}</h3>
+						{categories.map((item, index) => (
+							<li
+								key={`Category_${index}`}
+								className={
+									highlightedIndex === `Category_${index}`
+										? 'autosuggest-menu__item autosuggest-menu__item--highlighted'
+										: 'autosuggest-menu__item'
+								}
+								onClick={() => searchProduct(searchedKeyword, SuggestType.TopCategories, item)}
+								{...getInputProps({
+									onMouseOver: () => {
+										setKeyword(strip_html_tags(item.Value));
+									},
+								})}
+							>
+								<div dangerouslySetInnerHTML={{ __html: item.Value }} />
+							</li>
+						))}
+					</>
+				) : null}
+				{popular && popular.length ? (
+					<>
+						<h3>{PopularHeading}</h3>
+						{popular.map((item, index) => (
+							<li
+								key={`Popular_${index}`}
+								onClick={() => searchProduct(searchedKeyword, SuggestType.PopularSearches, item)}
+								className={
+									highlightedIndex === `Popular_${index}`
+										? 'autosuggest-menu__item autosuggest-menu__item--highlighted'
+										: 'autosuggest-menu__item'
+								}
+								{...getInputProps({
+									onMouseOver: () => {
+										setKeyword(strip_html_tags(item.Value));
+									},
+								})}
+							>
+								<div dangerouslySetInnerHTML={{ __html: item.Value }} />
+							</li>
+						))}
+					</>
+				) : null}
+				{content && content.length ? (
+					<>
+						<h3>{ContentHeading}</h3>
+						{content.map((item, index) => (
+							<li
+								key={`Content_${index}`}
+								onClick={() => searchProduct(searchedKeyword, SuggestType.TopContentMatches, item)}
+								className={
+									highlightedIndex === `Content_${index}`
+										? 'autosuggest-menu__item autosuggest-menu__item--highlighted'
+										: 'autosuggest-menu__item'
+								}
+								{...getInputProps({
+									onMouseOver: () => {
+										setKeyword(strip_html_tags(item.Value));
+									},
+								})}
+							>
+								{item.Value}
+							</li>
+						))}
+					</>
+				) : null}
+			</div>
+		</ul>
+	);
+}
+
+export default SuggestionList;
```

Comment:
I think the implementation should come from the Preview side, because the design may vary from client to client.

Reply:
The design should definitely be available for implementation by clients, but I think it is appropriate to have a basic one available. For instance, the ResultItem is always overridden, but there is a basic one available.